routes/auth.js

```diff
@@ -122,13 +127,14 @@ function usePassport() {
 }
 
 function getStrategy() {
+  console.log('getStrategy')
   return new GitHubStrategy(
     {
       clientID: options.clientId,
       clientSecret: options.clientSecret,
       // this needs to match the callback url on the oauth app on github
       callbackURL: `${endpoints.service}/auth/github/finalize`,
-      scope: ['public_repo', 'read:user', 'read:org']
+      scope: ['public_repo', 'read:user', 'read:org', 'user', 'user:email']
```

Comment:
Do we need both user and user:email scopes? Will the token's scopes need to be updated before deployment?

Reply:
The octokit documentation tells that both of them are required to get the emails.
Of course the token scope needs to be updated.

Reply:
It looks like there is no additional action for this OAuth app in GitHub settings.